ArmedForcesDB/1 table.sql

```diff
@@ -0,0 +1,25 @@
+use ArmedForcesDB
+go
+drop table if exists ArmedForces
+go
+create table dbo.ArmedForces(
+    ArmedForcesId int not null identity primary key,
+    SoldierFirstName varchar(30) not null constraint ck_ArmedForces_first_name_cannot_be_blank check(SoldierFirstName > ''),
+    SoldierLastName varchar(30) not null constraint ck_ArmedForces_last_name_cannot_be_blank check(SoldierLastName > ''),
+    SSN varchar(10) not null constraint ck_ArmedForces_ssn_must_be_9_or_10_digits_and_must_be_numeric check(SSN like replicate('[0-9]',9) or SSN like replicate('[0-9]',10)),
```

Comment:
The SSN should be inserted into the system in this structure: 8 numbers in a row and a dash before the check digit (the first digit on the right).